enforceBeachball.js

```diff
@@ -0,0 +1,12 @@
+const cp = require('child_process');
+
+cp.exec('yarn beachball < killProcessCommand.txt', (err, stdout, stderr) => {
+  if (!err && stdout.includes('No change files are needed')) {
+    console.log('Beachball guidelines were correctly followed. Continuing...');
+    return;
+  } else {
+    throw new Error(
+      "Change files should be created before merging. Please run 'yarn beachball' or yarn 'generate-change-files' from the monorepo root.",
```

Comment:
The script command "yarn generate-change-files" actually calls "yarn beachball change". Is "yarn beachball change" the same as "yarn beachball"? I feel like perhaps all our instructions should just include one command to run to keep it simple (e.g., "yarn generate-change-files").

Reply:
Good catch! I initially started out with `yarn beachball change` but slowly changed to `yarn beachball` because it's the same thing but shorter. I thought it would make more sense to use `yarn beachball` but in case people forgot the command I included an alias for it in the scripts- but now I'm thinking maybe that's not necessary. I'm going to replace the instances with `yarn changefile`. I think it's super clear what it's doing with that name and it's not much longer than `yarn beachball`.

Reply:
Yeah `changefile` is ok. `beachball:change` or `change` are the most common commands I've seen in repos

Reply:
Updated